media-sound/ario/ario-1.6-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools python-any-r1
+
+DESCRIPTION="yayy GTK client for MPD inspired by Rhythmbox but much lighter and faster"
+HOMEPAGE="http://ario-player.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}-player/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dbus debug +idle nls taglib zeroconf"
+
+RDEPEND=">=dev-libs/glib-2.39.4:2
+	dev-libs/libgcrypt:0=
+	dev-libs/libunique:1
+	>=dev-libs/libxml2-2.7.4:2
+	media-libs/libmpdclient
+	net-misc/curl
+	net-libs/gnutls
+	>=x11-libs/gtk+-3.21.5:3
+	dbus? ( dev-libs/dbus-glib )
+	x11-libs/libnotify
+	taglib? ( media-libs/taglib )
+	zeroconf? ( net-dns/avahi )"
+DEPEND="${RDEPEND}
+	dev-util/intltool
```

Comment:
This and the deps below belong to BDEPEND instead of DEPEND. Please sort this list, too.